lib/ir_security.ml

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
`ir_security.ml` and `ir_security.mli` are not used anywhere ... are you using them? if that's the case would be better to move this in the `krypto/` directory.